openvdb/Grid.h

```diff
@@ -501,6 +510,12 @@ class OPENVDB_API GridBase: public MetaMap
     /// @brief Initialize with an identity linear transform.
     GridBase(): mTransform(math::Transform::createLinearTransform()) {}
 
+#if OPENVDB_ABI_VERSION_NUMBER >= 7
+    /// @brief Initialize with metadata and a transform.
+    /// @throw ValueError if the transform pointer is null
```

Comment:
Add this `@throw` comment to the copy methods as well.

Reply:
Think I got them all. I didn't do the private Grid constructor as that appears to be the convention.

Reply:
I think you missed `GridBase::copyGrid`.

Reply:
Done, thanks.